i18n/en.json

```diff
@@ -2202,10 +2202,6 @@
     "id": "api.team.create_team_from_signup.expired_link.app_error",
     "translation": "The signup link has expired"
   },
-  {
-    "id": "api.team.create_team_from_signup.invalid_link.app_error",
-    "translation": "The signup link does not appear to be valid"
-  },
```

Comment:
There were a bunch of translation strings that all said the same thing and aren't actually used anywhere in the codebase. I removed them.